core/controllers/reader.py

```diff
@@ -81,6 +91,22 @@ def test_can_play(self, exploration_id, **kwargs):
 
 def _get_exploration_player_data(
         exploration_id, version, collection_id, can_edit):
+    """
+    This function is used to get exploration data which
```

Comment:
"Returns a dict containing exploration data."

(Maybe describe a bit more what 'exploration data' means, here or in the Returns section.